code/modules/admin/admin_verbs.dm

```diff
@@ -349,6 +355,54 @@ GLOBAL_PROTECT(admin_verbs_poll)
 			message_admins("[key_name_admin(usr)] checked antagonists.")
 	SSblackbox.record_feedback("tally", "admin_verb", 1, "Check Antagonists") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/list_bombers()
+	set name = "List Bombers"
+	set category = "Admin.Game"
+	if(!holder)
+		return
+	holder.list_bombers()
+	SSblackbox.record_feedback("tally", "admin_verb", 1, "List Bombers") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
```

Comment:
IF YOU ARE COPY-PASTING THIS ENSURE THE 2ND PARAMETER IS UNIQUE TO THE NEW PROC

Reply:
admin verbs are all together, it's fine

Reply:
okay but ```IF YOU ARE COPY-PASTING THIS ENSURE THE 2ND PARAMETER IS UNIQUE TO THE NEW PROC```